README.md

```diff
@@ -76,25 +76,53 @@ STRIPE_PUBLISHABLE_KEY=<replace-with-your-publishable-key>
 STRIPE_SECRET_KEY=<replace-with-your-secret-key>
 ```
 
-The other environment variables are configurable:
+**2. Create Products and Prices on Stripe**
 
-`STATIC_DIR` tells the server where to the client files are located and does not need to be modified unless you move the server files.
+This sample requires two [Price](https://stripe.com/docs/api/prices/object) IDs to create the Checkout page. Products and Prices are objects on Stripe that let you model a subscription.
 
-`BASIC_PRICE_ID` requires a Price ID for a "basic" subscription.
+**Using the Stripe CLI**
 
-`PRO_PRICE_ID` requires a Price ID for a "pro" subscription.
+```
+stripe products create --name="Premium" --description="Premium plan"
+```
 
-`DOMAIN` is the domain of your website, where Checkout will redirect back to after the customer completes the payment on the Checkout page.
+Create the price for the premium product, passing the product ID from the response:
 
-`CUSTOMER` is an ID for a Stripe Customer.
+```
+stripe prices create \
+  -d product=prod_XYZ \
+  -d unit_amount=1800 \
+  -d currency=usd \
+  -d "recurring[interval]"=month
+```
 
-**2. Create Products and Prices on Stripe**
+**Using the Dashboard**
 
-This sample requires two [Price](https://stripe.com/docs/api/prices/object) IDs to create the Checkout page. Products and Prices are objects on Stripe that let you model a subscription.
+You can create Products and Prices [in the dashboard](https://dashboard.stripe.com/products). Create two recurring Prices to run this sample.
+
+**Update BASIC_PRICE_ID and PRO_PRICE_ID in your .env file**
 
-You can create Products and Prices [in the dashboard](https://dashboard.stripe.com/products) or via [the API](https://stripe.com/docs/api/prices/create). Create two recurring Prices to run this sample.
+Repeat these steps for to create a second product and price.
 
-**3. Follow the server instructions on how to run:**
+Next, open `.env` in the folder of the server you want to use, and update the values for `BASIC_PRICE_ID` and  `PRO_PRICE_ID` with the price IDs of the two prices you added.
+
+**3. Create a customer**
```

Comment:
So we've waffled a bit on this point. (I forgot to remove from README). I don't think this variable is used anymore, right? Thinking we can remove this part?

Reply:
oh right! yep it is no longer used - revising!